pinot-core/src/main/java/org/apache/pinot/core/data/manager/BaseTableDataManager.java

```diff
@@ -122,15 +122,16 @@ public void init(TableDataManagerConfig tableDataManagerConfig, String instanceI
     _indexDir = new File(_tableDataDir);
     if (!_indexDir.exists()) {
       Preconditions.checkState(_indexDir.mkdirs(),
-          "Unable to create index directory at %s. Check that the user has permissions on this directory.", _indexDir);
+          "Unable to create index directory at %s. Check for available space and write-permissions for this directory.",
+          _indexDir);
     }
     _resourceTmpDir = new File(_indexDir, "tmp");
     // This is meant to cleanup temp resources from TableDataManager. But other code using this same
     // directory will have those deleted as well.
     FileUtils.deleteQuietly(_resourceTmpDir);
     if (!_resourceTmpDir.exists()) {
       Preconditions.checkState(_resourceTmpDir.mkdirs(),
-          "Unable to create temp resources directory at %s. Check that the user has permissions on this directory.",
+          "Unable to create temp resources directory at %s. Check for available space and write-permissions for this directory.",
```

Comment:
Please fix the length (limit of 120 character)

Reply:
I formatted it using IDE (and preferred formatting style). Somehow it chooses to keep the length.